api/src/integrations/epic/orders-utils.js

```diff
@@ -62,6 +62,41 @@ class Orders extends BaseRecordUtils {
       }
     };
   }
+
+  /**
+   * Indicates if a record is a fee order or not.
+   * 
+   * @param {object} transformedRecord Epic record that has been transformed to NRPTI format
+   * @returns {boolean} Indication if the record is a fee order
+   * @memberof Orders
+   */
+  isRecordFeeOrder(transformedRecord) {
+    if (!transformedRecord) {
+      throw new Error('isRecordFeeOrder - required transformedRecord must be non-null.');
+    }
+
+    // Only want to handle Orders.
+    if (transformedRecord.recordType !== 'Order' || !transformedRecord.recordName) {
```

Comment:
Logic question:

I would think `transformedRecord.recordType !== 'Order'` would result in a "false" from isRecordFeeOrder rather then throw an error? It's not necessarily an error case, just definitely not a fee order.

Reply:
This is being rewritten to address some other points too, so I don't think it will apply anymore.